tracer/build/_build/Build.VariableGenerations.cs

```diff
@@ -44,20 +54,52 @@ Target GenerateVariables
                        GenerateIntegrationTestsDebuggerArm64Matrices();
                    });
 
+            bool CriticalTracerChanges(string[] changedFiles, CodeOwnersParser codeOwners)
+            {
+                // These folders are owned by @DataDog/tracing-dotnet but changes should not affect ASM functionality
```

Comment:
I think this explanation is wrong?
```suggestion
                // These folders are owned by @DataDog/tracing-dotnet but changes could affect ASM functionality
```

Reply:
Actually, the folders listed are the ones that contain files that are not expected to affect ASM so much. I have considered that any change assigned to the tracer team should affect ASM by default unless the change is in this list.

For instance:
tracer/src/Datadog.Trace. -> Does not match the main tracer project files but it matches Datadog.Trace.BenchmarkDotNet, Datadog.Trace.Bundle, etc.
tracer/src/Datadog.Trace/OpenTelemetry/ -> Does not affect ASM